apps/frontend/src/components/global/groups/GroupManagement.vue

```diff
@@ -35,6 +35,29 @@
           />
         </v-container>
       </template>
+      <template #[`item.users`]="{item}">
+        <span v-for="(user, index) in item.users.slice(0, 4)" :key="user.id">
+          <v-chip
+            v-if="index < 3"
+            pill
+            color="primary"
+            class="ma-1"
+            @click="displayMembersDialog(item.users)"
+          >
+            {{ getMemberName(user, -1) }}
+          </v-chip>
+          <v-chip
+            v-else
+            pill
+            color="primary"
+            outlined
+            class="ma-1"
+            @click="displayMembersDialog(item.users)"
+          >
+            {{ `+${item.users.length} total` }}
+          </v-chip>
+        </span>
+      </template>
```

Comment:
Using the iterator here to display the +x total here feels a little wrong, is it possible to just do `item.users.slice(0, 3)` and then just put an if statement outside of the span but before the end of the template that shows `+x more`?